### PR TITLE
Fix list of default admission controllers to upstream defaults

### PR DESCRIPTION
**Pull request by dirkmueller**
See https://github.com/kubernetes/website/commit/0cc8cd3e3270c7569ad19d1f11791b8391972806#diff-507ecf555329a5a17ab732fcba09804a

## Why is this PR needed?

Does it fix an issue? addresses a business case?

add a description and link to the issue if one exists.

Fixes #

**Reminder**: Add the "fixes bsc#XXXX" to the title of the commit so that it will
appear in the changelog.

## What does this PR do?

please include a brief "management" technical overview (details are in the code)

## Anything else a reviewer needs to know?

Special test cases, manual steps, links to resources or anything else that could be helpful to the reviewer.

## Info for QA

This is info for QA so that they can validate this. This is **mandatory** if this PR fixes a bug.
If this is a new feature, a good description in "What does this PR do" may be enough.

### Related info

Info that can be relevant for QA:
* link to other PRs that should be merged together
* link to packages that should be released together
* upstream issues

### Status **BEFORE** applying the patch

How can we reproduce the issue? How can we see this issue? Please provide the steps and the prove
this issue is not fixed.

### Status **AFTER** applying the patch

How can we validate this issue is fixed? Please provide the steps and the prove this issue is fixed.

## Docs

If docs need to be updated, please add a link to a PR to https://github.com/SUSE/doc-caasp.
At the time of creating the issue, this PR can be work in progress (set its title to [WIP]),
but the documentation needs to be finalized before the PR can be merged. 

# Merge restrictions

(Please do not edit this)

We are in *v4-maintenance phase*, so we will restrict what can be merged to prevent unexpected surprises:

    What can be merged (merge criteria):
        2 approvals:
            1 developer: code is fine
            1 QA: QA is fine
        there is a PR for updating documentation (or a statement that this is not needed)

### CODE REVIEW

File: internal/pkg/skuba/kubeadm/configmap_test.go [hunk 1]
```diff
@@ -353,6 +353,19 @@ func TestUpdateClusterConfigurationWithClusterVersion(t *testing.T) {
...
+			expectedAdmissionPlugins: []string{
+				"NamespaceLifecycle", "LimitRanger", "ServiceAccount", "TaintNodesByCondition",
+				"Priority", "DefaultTolerationSeconds", "DefaultStorageClass",
+				"PersistentVolumeClaimResize", "MutatingAdmissionWebhook",
+				"ValidatingAdmissionWebhook", "ResourceQuota",
+				"StorageObjectInUseProtection", "RuntimeClass",
+				"CertificateApproval", "CertificateSigning", "CertificateSubjectRestriction",
+				"DefaultIngressClass", "NodeRestriction", "PodSecurityPolicy"},
+		},
 	}
```
Comment (cmurphy) [inline internal/pkg/skuba/kubeadm/configmap_test.go:369]: This needs to be rebased and adjusted to include `ExtendedResourceToleration` https://github.com/SUSE/skuba/pull/1192

**Comment by evrardjp** [comment]
It's on the X squad board to not fall off the radar, but I wonder if we shouldn't create a "Dirk squad one man army" label :laughing: , and assign it there. Joke aside, I will remove the X squad label, and ensure in the champions channel that this is handled by a team, whichever one is the more free.

File: internal/pkg/skuba/kubeadm/configmap_test.go (hunk 1 above)
Comment (Klaven) [reply internal/pkg/skuba/kubeadm/configmap_test.go:369]: https://github.com/SUSE/caasp-rfc/pull/62

**Comment by jordimassaguerpla** [comment]
I see this has the 5.0.0 label.
Is there an issue in github/SUSE/skuba for this?
We are in GMC phase, meaning only P1 issues should be included (P1=release blockers). Is this a P1 issue?

File: internal/pkg/skuba/kubeadm/configmap_test.go (hunk 1 above)
Comment (dirkmueller) [reply internal/pkg/skuba/kubeadm/configmap_test.go:369]: It has that already.

Comment (dirkmueller) [reply internal/pkg/skuba/kubeadm/configmap_test.go:369]: Rebased against 1.18.6, includes ExtendedResourceToleration now

**Comment by evrardjp** [comment]
This technically need rebase, to not try to migrate to 1.17.9 which was since then removed. Otherwise the code looks good _at first sight_.  I am testing this.

**Comment by evrardjp** [comment]
Works for me on upgrade.

**Comment by jordimassaguerpla** [comment]
Can this be released as a maintenance update? Or is there a reason to backport this to the release-caasp-4.5.0 branch? A reason would be that this is a P1 and so a release blocker.

**Comment by evrardjp** [comment]
I am not sure what are the product expectations in here, so I can't judge.

**Comment by evrardjp** [comment]
Ha, and here I was thinking that closing and reopening would trigger new tests...

**Review by chentex — APPROVED** [review]
LGTM

**Pull request merged**